includes/Embed_Block.php

```diff
@@ -145,11 +146,15 @@ public static function filter_kses_allowed_html( $allowed_tags ) {
 	 * @return string Rendered block type output.
 	 */
 	public function render_block( array $attributes, $content ) { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
-		// The only 2 mandatory attributes.
-		if ( empty( $attributes['url'] ) || empty( $attributes['title'] ) ) {
+		// The only mandatory attribute.
+		if ( empty( $attributes['url'] ) ) {
 			return '';
 		}
 
+		if ( empty( $attributes['title'] ) ) {
+			$attributes['title'] = __( 'Web Story', 'web-stories' );
```

Comment:
Needed because the `default` above does not kick in if the title is an empty string